relm4-macros/tests/builder.rs

```diff
@@ -2,10 +2,10 @@ use gtk::prelude::GtkWindowExt;
 use relm4::{gtk, ComponentParts, ComponentSender, SimpleComponent};
 
 #[derive(Default)]
-struct AppModel;
+struct App;
```

Comment:
I wonder if there's a place to put a "Component style guide" that has naming conventions like this in it.

Reply:
That's certainly possible, but requires us to create a custom `ItemImpl` because syn doesn't even parse this: https://github.com/Relm4/Relm4/blob/main/relm4-macros/src/lib.rs#L173

We cannot generate anything if we don't have an `ItemImpl` to place it inside.

Reply:
We could add something to the book. But I don't know whether that is actually necessary. It's only that the examples and tests should be consistent.

Reply:
Agreed. I'm just thinking back to when I started with Relm4 and didn't know what to call anything :)